portfolio/src/main/webapp/script.js

```diff
@@ -187,6 +187,7 @@ function addMultipleMessagesToDom(comments) {
 function getMessageFromJSON(pageReloadBoolean) {
   let selectedIndex;
   let amountSelector = document.getElementById(COMMENT_AMOUNT);
+
```

Comment:
Should this space be added?

Reply:
I think might have missed that. Thank you for catching it!

Removed, done!